### ISSUE TITLE
.fit doesn't print any loss value .

### ISSUE THREAD

**Issue by tempdeltavalue** [UKPLab/sentence-transformers#1498]
![Screenshot from 2022-04-03 12-59-04](https://user-images.githubusercontent.com/36921178/161422156-873bfc37-6575-41b5-a979-34df1aaa3602.png)

No information about that here..
https://www.sbert.net/docs/training/overview.html

**Comment by nreimers** [comment]
Print of loss function is not implemented

**Comment by tempdeltavalue** [comment]
Strange ...

**Comment by tempdeltavalue** [comment]
Any suggestion how to check loss or accuracy dynamically during training ?

**Comment by skullyhoofd** [comment]
@tempdeltavalue did you find a way to do this?

**Comment by tempdeltavalue** [comment]
@skullyhoofd No, according to comment @nreimers it's impossible :( (what is really strange)

In theory you can extract the model and train it using different .fit script but it's sounds like dangerous thing
